README.md

```diff
@@ -146,6 +146,11 @@ Environment Variables Supported:
 * MAILGUN_SECRET
 * MAILGUN_ADMIN_EMAIL
 * PORT
+* PGHOST (default: 'localhost')
+* PGUSER (default: process.env.USER)
+* PGPASSWORD (default: NULL)
```

Comment:
No need to show a default value for `PGPASSWORD` (but might want to add one for `PORT`...).